examples/webgl_renderer_pathtracer.html

```diff
@@ -0,0 +1,496 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title>three.js webgl - LDrawLoader</title>
```

Comment:
This title should be probably updated to something like `three.js webgl - path tracing`.